torchvision/models/detection/generalized_rcnn.py

```diff
@@ -77,6 +77,14 @@ def forward(self, images, targets=None):
             original_image_sizes.append((val[0], val[1]))
 
         images, targets = self.transform(images, targets)
+        if targets is not None:
+            for target in targets:
+                boxes = target["boxes"]
+                nonpostive_widths = (torch.sum(boxes[:, 2] <= boxes[:, 0]) > 0)
+                nonpostive_heights = (torch.sum(boxes[:, 3] <= boxes[:, 1]) > 0)
+                if nonpostive_widths > 0 or nonpostive_heights > 0:
```

Comment:
I believe this can be made a bit simpler (and more efficient) with `torch.any`:
```suggestion
                has_degenerate_boxes = torch.any(boxes[:, 2:] <= boxes[:, :2])
                if has_degenerate_boxes:
```